### ISSUE TITLE
iOS: Missing icon with Dev-working/latest

### ISSUE THREAD

**Issue by tildebyte** [hrydgard/ppsspp#8978]
I realize that this is kinda terse. I'm happy to provide more detail on request (I'm pretty knowledgeable in general, so terminal/iFile access for troubleshooting/data gathering are not a problem).
I don't when this first became an issue - I definitely haven't tried to play in week or so. I've been doing regular updates to the Dev-working builds.

iPhone 6s, 9.3.3 with JB
All builds are from cydia.ppsspp.org

Dev-working 0v1.2.2-971-g2adec71 (apparently synced with "latest"):
- Missing icon
 - Respring - not fixed
 - Clear uicache - not fixed
 - Reinstall - not fixed
 - Remove/install - not fixed

Replace with 1.1.1:
- Icon present, but FC on start (known issue? Probably not super-relevant in any case)

**Comment by hrydgard** [comment]
Thanks for reporting. Any idea what the problem might be? Pull requests accepted, I can't currently do any iOS work as I have no working iOS device at the moment (though I'm about to fix that situation).

**Comment by pablin1423** [comment]
Hi, thanks for the jit support in iOS ppsspp is the best app :) 
But I have the same issues like the above user even trying the same fixes.
Dev-latest/working update is only 10.1KB size. 
iPhone 6 and Ipad Mini 3 iOS 9.3.3

**Comment by tildebyte** [comment]
Shoot.
I don't even have an idea of where to start looking, but I've been googling around...

**Comment by tildebyte** [comment]
I didn't understand what @pablin1423 was saying, but: Good catch!
Definitely - after tapping "Install" for the dev builds, the "STATISTICS"/"Downloading" line shows "10.1 kB". The 1.1.1 shows 9.7 MB, so I assume it's a little short :)

**Comment by tildebyte** [comment]
Indeed, `PPSSPP.app` contains 1,270 bytes once installed.

**Comment by frankGucciLiberace** [comment]
Dude, just seems like a compilation error on the part of the builtbot. Deb from build.ppsspp.org is also looking a little light. 

Will probably be fine tmrw or whenever the bot recompiles

**Comment by tildebyte** [comment]
Cool. No good build yet, but I'm patient :)
Is there any way we can have builds available for direct d/l? It would have been pretty obvious that the build was bad...
Also, access to previous builds would be nice. It would make general troubleshooting easier, not least bisection.

**Comment by Darkjuda** [comment]
Got that issue too, my first guess was the buildbot not working properly.
I juste hope that doesn't erase the save files.

**Comment by frankGucciLiberace** [comment]
It won't, your saves (and games) are in the PSP directory most likely

**Comment by hrydgard** [comment]
This is related to Apples latest XCode update. Hopefully we can find a fix soon.

**Comment by hrydgard** [comment]
angelXwind has temporarily reverted the buildbot to the previous Xcode version. Did that fix the issue for now? We will need to properly fix the problem as well.

**Comment by Darkjuda** [comment]
It works. 1.2.2.973 now downloads properly (10.5 megs). The icon came back and working again.

**Comment by hrydgard** [comment]
Great, closing this (rather mistitled) issue for now.

**Comment by tildebyte** [comment]
@hrydgard I thought about changing the title, but if something like this happens again, an inexperienced end user *probably* won't search for "bad build" or some such.
Tags? Add "ridiculously small download" to the topic? :smile:

**Comment by Darkjuda** [comment]
Well, it's the contrary, at first I didn't saw this thread, so created a new one entitled "iOS buildbot might not be working properly". As soon as I found out there was already a thread about this issue, I closed mine.

I think the purpose of the titles is to sum up the issue so it becomes easier to understand so it can be fixed quickly, not by making it easier to understand by people who have no clue what they are doing here.
I'm saying that, but I am one of these people.
But if think if I don't even make the effort of understanding things like this, I have nothing to do here, I'll flood if anything else. 
But this is just my opinion.
